src/tag.h

```diff
@@ -16,9 +16,9 @@ class HSTag : public Object {
     HSTag(std::string name, Settings* settings);
     ~HSTag() override;
     std::shared_ptr<HSFrame>        frame;  // the master frame
-    Attribute_<unsigned long> index = {"index", 0};
-    Attribute_<bool>         floating = {"floating", false};
-    Attribute_<std::string>  name = {"name", {}};   // name of this tag
+    Attribute_<unsigned long> index;
+    Attribute_<bool>         floating;
+    Attribute_<std::string>  name;   // name of this tag
```

Comment:
While at it, could you fix or remove the member name indentation here? (I'd prefer removing it)

Reply:
What about the indentation in the constructor calls in the monitors.cpp file?

Reply:
Same. I don't see the benefit of that style, to be honest. (There are some very rare cases where I think otherwise, but this is not one of them.)

Reply:
The benefit is: it emphasizes their similarities and their differences. This means, that thanks to the indentation, I see directly, that `this` is passed to every attribute as the first parameter.

Reply:
Hold on, I was looking at the member declarations in monitor.h, not the ctor calls. Sorry. Okay, the ctor calls aren't that bad, so I actually *do* see the benefit there. I wouldn't do like that personally, but I also won't complain about it. ;-)